### PR TITLE
Renamed easy table to mobile table

### PR DESCRIPTION
**Pull request by soninaren**

### CODE REVIEW

File: Bindings/bindings.json [hunk 1]
```diff
@@ -1065,7 +1065,7 @@
         },
         {
             "type": "easyTable",
```
Comment (mathewc) [inline Bindings/bindings.json]: why is the type still easyTable? We've already released the breaking change to rename that to mobileTable in the runtime.

**Comment by soninaren** [comment]
updated.

**Comment by brettsam** [comment]
This looks good to me. Let's wait until 0.2 is the default version, then get this in.

**Comment by soninaren** [comment]
This is merging into dev, Will merge dev to master once we have 0.2 as default.

**Pull request merged**